src/android/jni/jni_logging.h

```diff
@@ -0,0 +1,16 @@
+/* License: Apache 2.0. See LICENSE file in root directory. */
+/* Copyright(c) 2021 Intel Corporation. All Rights Reserved. */
+
+#ifndef LIBREALSENSE_JNI_LOGGING_H
+#define LIBREALSENSE_JNI_LOGGING_H
+
+#include <android/log.h>
+
+#define  LRS_JNI_LOG_TAG    "librs jni api"
```

Comment:
I'd rather the tag to be one word, in order to make it easier to work with.
If it is ok for you, please change it to librs_jni_api (or librs_jni, maybe better)

Reply:
fixed.